tools/monorepo-utils/src/code-freeze/commands/version-bump/lib/validate.ts

```diff
@@ -10,6 +10,20 @@ import { readFile } from 'fs/promises';
  */
 import { Logger } from '../../../../core/logger';
 import { Options } from '../types';
+
+/**
+ * Determine whether a version is an accel release.
+ *
+ * @param {string} version Version number
+ * @return {bool} True if the version corresponds with an accel release, otherwise false
+ */
+export const getIsAccelRelease = (
+	version: string
+): boolean => {
+	const isAccelRelease = version.match( /^(?:\d+\.){3}\d+(?:\.\d+)?$/ );
```

Comment:
This regex will also match 5 digits -- 6.0.0.10 is okay, but 6.0.0.0.10 is also okay.  If you don't want it to be able to handle 5 digits, it should be /^(?:\d+\.){2}\d+(?:\.\d+)?$/ -- but maybe we want 4 and 5 to be okay in case we ever need to do an AF point release?  I don't know?

Reply:
Thanks, good catch... I think I originally wrote this before we decided for sure on everything being 4-digit. Updated!